GDevelop.js/Bindings/Bindings.idl

```diff
@@ -2394,6 +2394,7 @@ interface ParticleEmitterObject {
     void SetRenderingAdditive();
     void SetRenderingAlpha();
 
+    float GetNbParticles();
```

Comment:
Same remark as before, this is actually not needed, because we don't use this class in the game engine. This is "just" the object for the IDE - which does not care how many particles exist because it's not running the game. Only the TypeScript game engine has this information, thanks to `getParticleCount`